### PR TITLE
Adds Particle Network to list of wallets

### PR DESCRIPTION
**Pull request by TABASCOatw**
Adds Particle Network to list of wallets in `getting-started.md`.

Additionally, this PR adds a page within the wallet section for Particle Network, modeled closely after existing pages in the same section.

### CODE REVIEW

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thanks for the contribution @TABASCOatw, we are reviewing.

In the meantime, do you mind adding the document ID to `sidebars.js`? Would recommend adding it [here](https://github.com/0xPolygon/wiki/blob/main/sidebars.js#L106C15-L106C15).

**Comment by TABASCOatw** [comment]
Sure, np, just made the change.

File: docs/tools/wallets/getting-started.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ The following wallets are solutions that Polygon Technology has partnered with:
 | [Wirex](https://wirexapp.com/en/wirex-wallet)* 	| non-custodial 	| EOA 	| yes 	| no 	| no 	|  	|  	| mobile 	|
 | [XDeFi](https://www.xdefi.io/) 	| non-custodial 	| EOA 	| no 	| interface 	| no 	| no 	| no 	| browser 	|
 | [Zerion](https://zerion.io/) 	| non-custodial 	| EOA 	| no 	| yes 	| yes 	|yes|  	| mobile, browser 	|
+| [Particle Network](https://particle.network) 	| non-custodial  | hybrid 	| no 	| yes 	| yes 	| yes 	| yes 	| mobile, browser, api/sdk 	|
```
Comment (AP1400) [inline docs/tools/wallets/getting-started.md:118]: Get back on metamask wallet

**Review by AP1400 — CHANGES_REQUESTED** [review]
What should I do

File: docs/tools/wallets/getting-started.md (hunk 1 above)
Comment (TABASCOatw) [reply docs/tools/wallets/getting-started.md:118]: Hey @AP1400, I'm not entirely sure what you mean. Could you clarify?

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**